src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
We won't need this just yet. You can remove this

Reply:
can you please tell me why we don't need service worker? Based on what I read [here](https://stackoverflow.com/questions/49314386/what-is-service-worker-in-react-js) it seems like we're gonna use it a lot, especially for the following features:
```
* ability to intercept and handle network requests
* programmatically managing a cache of responses
* helps cache assets and other files when the user is offline or on slow network
```

I understand we don't need the service worker for this PR specifically, but why do we have to limit ourselves to only this PR? Isn't the purpose of using create-react-app boilerplate is so that we can get the basic stuff it provides that is enough to get us started instead of writing everything from scratch?